app/signup/route.js

```diff
@@ -20,8 +27,8 @@ export default Ember.Route.extend({
         // TODO: This option causes a user request to fetch data already present in the identity map
         return session.open('aptible', credentials);
       }).then(function(){
-        route.transitionTo('index');
-      });
+        replaceLocation([config.legacyDashboardHost, 'organizations/new'].join('/'));
+      }, function(){});
```

Comment:
without a comment explaining that `session.open` failures are ok to be ignored (I assume that's the intent?), this empty function is a little confusing.